test/integration/extensions/controller/backupbucket/backupbucket_test.go

```diff
@@ -19,17 +19,17 @@ import (
 	"fmt"
 	"time"
 
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
TBH: I don't want to sort all these imports manually, as it doesn't really matter.
If you think, we need to care about import sorting, please open an issue to add a tool to enforce and automatically sort the imports according to our preferences.
There is simply no point in commenting about this and manually sorting, because it is a task for the machines and wastes time for us as humans.